ivy/functional/frontends/paddle/tensor/math.py

```diff
@@ -319,6 +319,12 @@ def trunc(x, name=None):
     return ivy.trunc(x)
 
 
+@with_supported_dtypes({"2.5.0 and below": ("float32", "float64")}, "paddle")
+@to_ivy_arrays_and_back
+def digamma(x, name=None):
+    return ivy.digamma(x)
```

Comment:
This function isn't implemented on the functional API you'll need to write a composition🙂

Reply:
Hi @hirwa-nshuti. After investigating further, I found that Ivy currently lacks the gamma function and its derivative, which are required for the digamma function. I would appreciate your guidance on how to proceed with implementing the digamma function in Ivy. Thank you for your support.